engine/src/main/java/org/terasology/rendering/dag/nodes/ChunksRefractiveReflectiveNode.java

```diff
@@ -97,4 +102,11 @@ private void postRenderCleanupSceneReflectiveRefractive() {
         }
         bindDisplay();
     }
+
+    @Override
+    public void update() {
+        // TODO: renames, maybe?
+        FBO sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
+        sceneOpaque.attachDepthBufferTo(dynamicFBM.getFBO(REFRACTIVE_REFLECTIVE_URN));
```

Comment:
Hmmm... I feel this might be ok short term but long term it might be problematic. I.e. what if this node gets replaced by a module having a more advanced version of it but _NOT_ re-attaching the depth buffer to the refr/refl FBO?

One way to handle this could be to have our first NON-rendering node, one whose only job is to be in the DAG only if the reflective/refractive FBO is needed and making sure the current ReadOnlyGBuffer and the refl/refr FBO share the depth buffer, reacting to the FBOs being regenerated.

Alternatively, the FBO.attachDepthBufferTo() method should track what is attached to what so that it doesn't reattach anything that is already attached. This way all nodes can use the method safely.

I'd rather prefer the first option though. Feels neater. Perhaps you could write a TODO about this? Perhaps in WorldRendererImpl, near the reflection node?